packages/react-native-editor/ios/Podfile.lock

```diff
@@ -416,9 +416,9 @@ SPEC CHECKSUMS:
   react-native-get-random-values: 03edb8dcc2d3f43e55aa67ea13b61b6723bbf047
   react-native-keyboard-aware-scroll-view: 6cb84879bf07e4cc1caed18b11fb928e142adac6
   react-native-safe-area: c9cf765aa2dd96159476a99633e7d462ce5bb94f
-  react-native-safe-area-context: c8da723dcddabe15afe95c9d31c56a0cf2b0141c
+  react-native-safe-area-context: 133d77d86b13cf694e014ab03dc2944ee3125d97
```

Comment:
I wonder why all of these references are being updated, they should be synced with what we have in `master` since nothing on this PR is updating them. Maybe this branch is not updated with the main branch?

Reply:
@geriux the changes to `react-native-safe-area-context` and a few others were likely artifacts from building/testing while rebasing. I have reverted those changes.

The changes of `Gutenberg` and `RNTAztecView` continue to occur when I build with my latest work, though. Is that expected? If not, any thoughts on why this may be occurring?

```diff
diff --git a/packages/react-native-editor/ios/Podfile.lock b/packages/react-native-editor/ios/Podfile.lock
index a5e8a6539c..73265c45d4 100644
--- a/packages/react-native-editor/ios/Podfile.lock
+++ b/packages/react-native-editor/ios/Podfile.lock
@@ -402,7 +402,7 @@ SPEC CHECKSUMS:
   FBReactNativeSpec: 118d0d177724c2d67f08a59136eb29ef5943ec75
   Folly: 30e7936e1c45c08d884aa59369ed951a8e68cf51
   glog: 1f3da668190260b06b429bb211bfbee5cd790c28
-  Gutenberg: c1ffe7f143c35f6d19a4ed0011b78019d11fe03b
+  Gutenberg: 85ac96cd7a5e81b9c1307a0fbe979df0a8e5c45d
   RCTRequired: b153add4da6e7dbc44aebf93f3cf4fcae392ddf1
   RCTTypeSafety: 9aa1b91d7f9310fc6eadc3cf95126ffe818af320
   React: b6a59ef847b2b40bb6e0180a97d0ca716969ac78
@@ -435,7 +435,7 @@ SPEC CHECKSUMS:
   RNReanimated: f05baf4cd76b6eab2e4d7e2b244424960b968918
   RNScreens: 953633729a42e23ad0c93574d676b361e3335e8b
   RNSVG: 46c4b680fe18237fa01eb7d7b311d77618fde31f
-  RNTAztecView: 14e03d6cc8719ad21fbbc0678f887fcefa9ea42b
+  RNTAztecView: 79a4e1a94eaae797b6cf045c55cc97cde205efe6
   WordPress-Aztec-iOS: b7ac8b30f746992e85d9668453ac87c2cdcecf4f
   Yoga: f2a7cd4280bfe2cca5a7aed98ba0eb3d1310f18b
```

Reply:
I've just checked and it is happening on the main branch as well, running `npm run native ios` updates the `Podfile.lock` this file is mostly updated when we make a new release. Since we are cutting a release today, once that's merged it'll solve these changes.